modules/ROOT/pages/dataweave-formats-xml.adoc

```diff
@@ -23,7 +23,26 @@ To understand the parsing strategies that DataWeave readers and writers can appl
 [[format_xml_cdata]]
 == CData Custom Type
 
-`CData` is a custom DataWeave data type for XML that is used to identify a Character DATA (CData) block. The `CData` type tells the XML writer to wrap content inside a `CData` block or to check for an input string inside a `CData` block. In DataWeave, `CData` inherits from the type `String`.
+`CData` is a custom DataWeave data type for XML that is used to identify a character data (CDATA) block. The `CData` type makes the XML writer wrap content inside a `CDATA` block or to check for an input string inside a `CDATA` block. In DataWeave, `CData` inherits from the type `String`.
+
+== Doctype Custom Type
+
+`DocType` is a custom DataWeave data type for XML that is used to identify a doctype declaration.
+
+== Read and Write DTDs
+
+_Introduced in DataWeave 2.5.0. Supported by Mule 4.5.0 and later._
+
+DataWeave can read and write doctype declarations in XML files.
+
+To read a doctype declarations, set the xref:dataweave-system-properties.adoc[system property] `com.mulesoft.dw.xml_reader.parseDtd`.
```

Comment:
TODO: add info to what's new on new features

Reply:
added note about this to https://github.com/mulesoft/docs-dataweave/pull/222/files